build errors when Jsonm is not installed but other external deps are

```
===== ERROR while installing opam-lib.1.2.0 =====
# opam-version 1.1.2-rc3
# os           linux
# command      make
# path         /home/travis/.opam/system/build/opam-lib.1.2.0
# compiler     system (4.02.1)
# exit-code    2
# env-file     /home/travis/.opam/system/build/opam-lib.1.2.0/opam-lib-13301-ad6566.env
# stdout-file  /home/travis/.opam/system/build/opam-lib.1.2.0/opam-lib-13301-ad6566.out
# stderr-file  /home/travis/.opam/system/build/opam-lib.1.2.0/opam-lib-13301-ad6566.err
### stdout ###
# ...[truncated]
# ocamlfind ocamldep -I core -I solver -I repositories -I client -I core/ core/opamMisc.ml > ._d/core/opamMisc.d
# ocamlfind ocamldep -I core -I solver -I repositories -I client -I core/ core/opamJson.ml > ._d/core/opamJson.d
# make[5]: Leaving directory `/home/travis/.opam/system/build/opam-lib.1.2.0/src'
# make[5]: Entering directory `/home/travis/.opam/system/build/opam-lib.1.2.0/src'
# ocamlfind ocamlc -package unix,extlib,re,re.emacs,re.str,re.perl,re.pcre,re.glob,cmdliner,ocamlgraph,cudf,dose3,no -c -g -w +a-4-9-32-41-44-45 -bin-annot -short-paths -I core -I solver -I repositories -I client -I core/ core/opamJson.mli
# make[5]: Leaving directory `/home/travis/.opam/system/build/opam-lib.1.2.0/src'
# make[4]: Leaving directory `/home/travis/.opam/system/build/opam-lib.1.2.0/src'
# make[3]: Leaving directory `/home/travis/.opam/system/build/opam-lib.1.2.0/src'
# make[2]: Leaving directory `/home/travis/.opam/system/build/opam-lib.1.2.0/src'
# make[1]: Leaving directory `/home/travis/.opam/system/build/opam-lib.1.2.0/src'
### stderr ###
# ocamlfind: Package `no' not found
# make[5]: *** [core/opamJson.cmi] Error 2
# make[4]: *** [byte-code-library] Error 2
# make[3]: *** [subproj_core] Error 2
# make[2]: *** [opam-core.cma] Error 2
# make[1]: *** [opam-lib.byte] Error 2
# make: *** [opam-lib] Error 2
```

Seems that there are still some `sed` magic hanging around ...

--- a/configure
+++ b/configure
@@ -587,7 +587,6 @@ ac_subst_vars='LTLIBOBJS
 LIBOBJS
 hasalldeps
 OCAML_PKG_jsonm
-OCAML_PKG_uutf
 OCAML_PKG_dose3
 OCAML_PKG_cudf
 OCAML_PKG_ocamlgraph
@@ -4477,30 +4476,6 @@ $as_echo "not found" >&6; }
 
 
 
-  { $as_echo "$as_me:${as_lineno-$LINENO}: checking for OCaml findlib package uutf" >&5
-$as_echo_n "checking for OCaml findlib package uutf... " >&6; }
-
-  unset found
-  unset pkg
-  found=no
-  for pkg in uutf  ; do
-    if $OCAMLFIND query $pkg >/dev/null 2>/dev/null; then
-      { $as_echo "$as_me:${as_lineno-$LINENO}: result: found" >&5
-$as_echo "found" >&6; }
-      OCAML_PKG_uutf=$pkg
-      found=yes
-      break
-    fi
-  done
-  if test "$found" = "no" ; then
-    { $as_echo "$as_me:${as_lineno-$LINENO}: result: not found" >&5
-$as_echo "not found" >&6; }
-    OCAML_PKG_uutf=no
-  fi
-
-
-
-
   { $as_echo "$as_me:${as_lineno-$LINENO}: checking for OCaml findlib package jsonm" >&5
 $as_echo_n "checking for OCaml findlib package jsonm... " >&6; }
 
@@ -4532,7 +4507,8 @@ if test "$OCAML_PKG_extlib" = "no" ||
    test "$OCAML_PKG_cmdliner" = "no" ||
    test "$OCAML_PKG_ocamlgraph" = "no" ||
    test "$OCAML_PKG_cudf" = "no" ||
-   test "$OCAML_PKG_dose3" = "no"; then
+   test "$OCAML_PKG_dose3" = "no" ||
+   test "$OCAML_PKG_jsonm" = "no"; then
   echo "============================================================================"
   echo "Some dependencies are missing. If you are just interested in the stand-alone"
   echo "binaries, run 'make lib-ext' to download and include them."

--- a/configure.ac
+++ b/configure.ac
@@ -73,6 +73,7 @@ dnl echo "cmdliner...................... ${OCAML_PKG_cmdliner}"
 dnl echo "graph......................... ${OCAML_PKG_ocamlgraph}"
 dnl echo "cudf.......................... ${OCAML_PKG_cudf}"
 dnl echo "dose3......................... ${OCAML_PKG_dose3}"
+dnl echo "jsonm......................... ${OCAML_PKG_jsonm}"
 echo
 
 if test "$OCAML_PKG_extlib" = "no" ||
@@ -80,7 +81,8 @@ if test "$OCAML_PKG_extlib" = "no" ||
    test "$OCAML_PKG_cmdliner" = "no" ||
    test "$OCAML_PKG_ocamlgraph" = "no" ||
    test "$OCAML_PKG_cudf" = "no" ||
-   test "$OCAML_PKG_dose3" = "no"; then
+   test "$OCAML_PKG_dose3" = "no" ||
+   test "$OCAML_PKG_jsonm" = "no"; then
   echo "============================================================================"
   echo "Some dependencies are missing. If you are just interested in the stand-alone"
   echo "binaries, run 'make lib-ext' to download and include them."

